activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/swagger/SwaggerConfiguration.java

```diff
@@ -16,60 +16,22 @@
 
 package org.activiti.cloud.services.organization.swagger;
 
-import org.activiti.cloud.alfresco.rest.model.ListResponseContent;
-import org.activiti.cloud.organization.api.Project;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import springfox.documentation.builders.ApiInfoBuilder;
-import springfox.documentation.builders.PathSelectors;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.service.ApiInfo;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger.web.UiConfiguration;
 import springfox.documentation.swagger.web.UiConfigurationBuilder;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 /**
  * Swagger configuration
  */
 @Configuration
-@EnableSwagger2
 public class SwaggerConfiguration {
 
-    @Bean
-    @ConditionalOnMissingBean(Docket.class)
-    public Docket api() {
-        return new Docket(DocumentationType.SWAGGER_2)
-                .select()
-                .apis(RequestHandlerSelectors.basePackage("org.activiti.cloud.services.organization.rest.controller"))
-                .paths(PathSelectors.any())
-                .build()
-                .apiInfo(apiInfo());
-    }
-
-    @Bean
-    public ApiInfo apiInfo() {
-        return new ApiInfoBuilder()
-                .title("Activiti Modeling REST API")
-                .description("Provides access to the core features of Activiti Modeling.")
-                .version("1.0")
-                .build();
-    }
-
     @Bean
```

Comment:
I don't think we really need to define this bean. I think it was experimentally done for showing the call duration.

Reply:
Ok. So I'll remove it.